### ISSUE TITLE
Question about "compute correctness loss"

### ISSUE THREAD

**Issue by RingoTC** [lucy66666/OKT#1]
You did excellent work!
But I have a question about the implementation of computing correctness loss.
<img width="890" alt="image" src="https://user-images.githubusercontent.com/6525061/210303135-08773127-2406-4076-87fa-0dc74a27b016.png">
I'm confused that the loss is assigned to loss_cls which makes loss_cls have no relation to the former computations like logits.

**Comment by lucy66666** [comment]
We forgot to uncomment the line earlier and have resolved the bug! Thanks for pointing it out!!!
